### DISCUSSION TITLE
can i use this plugin to fully control my browser?

### DISCUSSION THREAD

**Discussion by cStralpt** [glacambre/firenvim#1569] [Q&A]
i want to control my browser fully using a keyboard, is this plugin gonna work like that?
for example: doing scroll using `<C-d/e/u/y>`, switching buffer(browser tabs) using `<Shift-h/l>`, and any other keymaps.
i ask this because because on the readme demo i only see use case for text editing only.

**Comment by alerque** [comment]
No, this plugin is just for editing text, not controlling the browser.

I suggest [Tridactyl](https://tridactyl.xyz/about/) which allows browser interaction to be *almost* completely keyboard based and uses VIM-like paradigms including modality and some composability in the bindings. You can (as I do) additionally use Firenvim for the text editing features.

**Comment by glacambre** [comment]
Hi,

No, Firenvim is exclusively for text editing. If you want to control your browser using vim-like keybindings, I would recommend using Tridactyl instead, it's what I use and I'm pretty happy with it :).
